docs/news.rst

```diff
@@ -64,7 +64,7 @@ New features
     provides a cleaner way to pass keyword arguments to callback methods
     (:issue:`1138`, :issue:`3563`)
 
-*   A new :class:`~scrapy.http.JSONRequest` class offers a more convenient way
+*   A new :class:`~scrapy.http.JsonRequest` class offers a more convenient way
```

Comment:
To comply with @elacuesta’s feedback while keeping a working link:

```suggestion
*   A new :class:`JSONRequest <scrapy.http.JsonRequest>` class offers a more convenient way
```

Reply:
Tested in local and it works, thanks :slightly_smiling_face: